File: AUtil/app/src/main/java/com/example/administrator/util/L.java
package com.example.administrator.util;

import android.util.Log;

/**
 * Created by 舍长 on 2018/2/22.
 * 描述:Log设置类
 * 引用至:https://www.jianshu.com/p/6784800b5cc9
 */

public class L {

    //TAG
    public static String TAG = "tonjies";

    //5个等级 DIWE

    public static void d(String text) {
        Log.d(TAG, text + "");
    }

    public static void i(String text) {
        Log.i(TAG, text + "");
    }

    public static void w(String text) {
        Log.w(TAG, text + "");
    }

    public static void e(String text) {
        Log.e(TAG, text + "");
    }

}

File: kotlinPractice2/app/src/main/java/com/example/tonjies/kotlinpractice2/util/L.kt
package com.example.tonjies.kotlinpractice2.util

import android.util.Log

/**
 * 在kotlin中，加了object后，L类就成为了一个单例模式的类，相当于帮我们省略掉了以前Java实现单例的代码
 * 最后我们可以直接L.d调用类中的方法
 */
object L {
    //过滤词
    val TAG:String="tonjies"
    //5个等级 DIWE

    fun d(text: String) {
        Log.d(TAG, text + "")
    }
}